src/Components/Authentication/Mobile/SignUpForm.tsx

```diff
@@ -18,17 +20,36 @@ import { ProgressIndicator } from "Components/ProgressIndicator"
 import { Step, Wizard } from "Components/Wizard"
 import React, { Component, Fragment } from "react"
 
+export interface Props extends FormProps {
+  contextModule?: string
+}
+
 export interface MobileSignUpFormState {
   isSocialSignUp: boolean
 }
-export class MobileSignUpForm extends Component<
-  FormProps,
-  MobileSignUpFormState
-> {
+
+export const currentStepActionName = {
+  0: Schema.ActionName.EmailNextButton,
+  1: Schema.ActionName.PasswordNextButton,
+}
+
+@track({ context_module: Schema.Context.Header })
+export class MobileSignUpForm extends Component<Props, MobileSignUpFormState> {
   state = {
     isSocialSignUp: false,
   }
 
+  @track((props, state, args) => ({
```

Comment:
I pass the `currentStepIndex` to the `trackNextClick` and access it via the args here.